epub33/explainers/EPUB-33-security-privacy.md

```diff
@@ -0,0 +1,103 @@
+
+EPUB Reading Systems typically collect information on the reading habits of their users. In some cases this information is necessary to provide a good user experience; in other cases the information is collected for other users—the reading system company or the publisher of the book. 
+
+1. Tracking reading position. You really do want the reading system to remember that you just started chapter 5. 
+2. Remembering highlights, notes, bookmarks. 
+3. Remembering user preferences around fonts, visual themes (including dark/light), text justification, etc. 
+4. Recording progress through a book and whether a book was finished. 
+5. Some reading systems track every user reading session, including the time of day, the duration, how many pages were read, what book, the user’s IP address if a web-based reader, etc. 
+
+
+
+### 2. Is this specification exposing the minimum amount of information necessary to power the feature?
+
+The EPUB specification does not mention reading system features like bookmarks or highlights. There is no requirement to remember reading state. 
+
+### 3. How does this specification deal with personal information or personally-identifiable information or information derived thereof?
+
+The EPUB specifications do not mention personal information.
+
+### 4. How does this specification deal with sensitive information?
+
+Merely the fact that you are reading a particular book can be sensitive information. Again, the specification offers no guidance to reading systems on this matter. 
+
+### 5. Does this specification introduce new state for an origin that persists across browsing sessions?
+
+Reading systems, in order to meet user expectations, must retain some information across browsing sessions, for example to remember reading location. The spec has a non-normative section saying that reading systems should treat each EPUB as a unique origin, but does not say anything about most reading system behaviors.
+
+### 6. What information from the underlying platform, e.g. configuration data, is exposed by this specification to an origin?
+
+Many EPUB reading systems do not support scripting at all. For those that do, scripts in EPUBs that seek this sort of information might be blocked by the application before they even reach the platform level. If not, the risks are likely similar to that of HTML/CSS. 
+
+
+### 7. Does this specification allow an origin access to sensors on a user’s device?
+
+EPUB does have requirements for user agents to respond to device orientation (`portrait | landscape`) for fixed-layout EPUBs. 
+
+There has been talk about scripted EPUBs using geolocation information; I don’t know if this has happened in the wild yet, or if any reading systems would allow access to this information. This would happen through existing Web APIs; EPUB does not specify new APIs to interact with sensors. 
+
```

Comment:
I do not think we should go into these details. EPUB does not include any extra access to sensors beyond the Web APIs. That is it.

Reply:
It is actually important to stress this elsewhere (maybe in the intro): EPUB does _not_ create _any_ new API, access method, etc. It relies on existing Web API-s. Ie (and that is important to emphasize) _EPUB does not fork the Web!_

Reply:
We do have the `epubReadingSystem` object...

Reply:
... which we may turn into optional:-) But you are right, it is a minor addition to the existing structures. We can add something like "(except for single entry to get back an identification of the RS)". I wish all Web API-s were as simple:-)

Reply:
> But EPUB does allow scripting, and scripting might allow access to sensors

... via API-s that are defined on the Open Web Platform. The important point is that we do not introduce anything new in this respect. Selfishly: this also means that the related security/privacy problems are not for us to solve...

Reply:
I already have 

> This would happen through existing Web APIs; EPUB does not specify new APIs to interact with sensors. 

Is this not enough?

Reply:
I missed that... sorry.